pkg/util/v1alpha3/katibclient/katib_client.go

```diff
@@ -150,7 +151,10 @@ func (k *KatibClient) GetTrialTemplates(namespace ...string) (map[string]string,
 	ns := getNamespace(namespace...)
 	trialTemplates := &apiv1.ConfigMap{}
 
-	if err := k.client.Get(context.Background(), types.NamespacedName{Name: experimentsv1alpha3.DefaultTrialConfigMapName, Namespace: ns}, trialTemplates); err != nil {
+	err := k.client.Get(context.TODO(), types.NamespacedName{Name: experimentsv1alpha3.DefaultTrialConfigMapName, Namespace: ns}, trialTemplates)
```

Comment:
`return k.GetConfigMap(experimentsv1alpha3.DefaultTrialConfigMapName, namespace)`

BTW, now trialTemplate can be stored in any configmap, should we always get/update it in default ConfigMap?

Reply:
I can refactor it in another PR. Now #774 is in higher priority.

Reply:
sure